var/spack/repos/builtin/packages/protobuf/package.py

```diff
@@ -112,7 +112,7 @@ class Protobuf(CMakePackage):
     patch(
         "https://github.com/protocolbuffers/protobuf/pull/11032/commits/3039f932aaf212bcf2f14a3f2fd00dbfb881e46b.patch?full_index=1",
```

Comment:
Per discussion in https://github.com/spack/spack/pull/44212#issuecomment-2118169017:
```suggestion
        "https://github.com/protocolbuffers/protobuf/commit/3039f932aaf212bcf2f14a3f2fd00dbfb881e46b.patch?full_index=1",
```